app/src/main/res/values/strings.xml

```diff
@@ -117,36 +117,28 @@
         like better. With enough comparison picks, the app will create a ranking from whom you like
         the most to least.\n
         \n
-        Next the app limits every user to maximum 10 matches. Matches are made based on whom you
+        Next the app limits every user to maximum 10 matches. You get matched based on whom you
         like the most and who likes you back. If you both are super popular, chances are you\'ll get
-        matched up unless either of you runs out of matches. If you are more average attractive,
-        chances are that your top choices will run out of matches before you\'d even get considered.
-        You\'ll likely end up matching with other average attractive people who couldn\'t get their
-        first choices.\n
+        matched up unless either of you runs out of matches.\n
```

Comment:
I like the direction of shortening the text. Yet the text seems to jump around topics too fast without explaining the user enough to understand what's going on.

What does it mean to run out of matches? That probably needs a bit more context.

The way how the "niche" is introduced wouldn't make sense to me. You kind of have the words and phrases there, yet to the uninitiated, it doesn't seem to introduce it in a comprehensible way.